therapi-runtime-javadoc/src/main/java/com/github/therapi/runtimejavadoc/MethodJavadoc.java

```diff
@@ -16,17 +16,25 @@
 
 package com.github.therapi.runtimejavadoc;
 
+import com.github.therapi.runtimejavadoc.internal.MethodJavadocKey;
+import static com.github.therapi.runtimejavadoc.internal.RuntimeJavadocHelper.unmodifiableDefensiveCopy;
 import java.lang.reflect.Constructor;
+import java.lang.reflect.Executable;
 import java.lang.reflect.Method;
 import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.LinkedHashMap;
 import java.util.List;
-
-import static com.github.therapi.runtimejavadoc.internal.RuntimeJavadocHelper.unmodifiableDefensiveCopy;
+import java.util.Map;
+import java.util.stream.Collectors;
 
 public class MethodJavadoc extends BaseJavadoc {
+    public static final String INIT = "<init>";
```

Comment:
I'd prefer for the `INIT` constant to not be part of the public API. Please can you move it to `RuntimeJavadocHelper`?

Reply:
Done